R/driver.R

```diff
@@ -137,7 +148,8 @@ driver <- function(all_data = empty_data(), write_outputs = TRUE, quiet = FALSE,
       if(!quiet) print(paste("- make", format(round(tdiff, 2), nsmall = 2)))
 
       check_chunk_outputs(chunk, chunk_data, input_names,
-                          promised_outputs = subset(chunkoutputs, name == chunk)$output)
+                          promised_outputs = subset(chunkoutputs, name == chunk)$output,
+                          outputs_xml = subset(chunkoutputs, name == chunk)$to_xml)
```

Comment:
No, they are declared at the same time.  It is just that some outputs may be tagged as XML.  I don't think we have an instance where this would happen but a hypothetical example is:
```
  } else if(command == driver.DECLARE_OUTPUTS) {
    return(c("L200.ModelTimeInterYears_adjusted", XML = "modeltime.xml"))
  } else if(command == driver.MAKE) {
```